_includes/footer.html

```diff
@@ -0,0 +1,3 @@
+<footer>
+  <p>&copy; University of Waterloo 2018 &mdash; 2019.</p>
```

Comment:
<img width="308" alt="screen shot 2018-09-03 at 2 14 10 pm" src="https://user-images.githubusercontent.com/15850091/45001684-089f4680-af84-11e8-846a-04f071a3cfcb.png">
was wondering if the long dash here is intended or not, no strong opinion on this though.

Reply:
is intended, but style is debatable.